src/acktest/bootstrapping/vpc.py

```diff
@@ -144,6 +144,50 @@ def cleanup(self):
     def get_availability_zone_names(self):
         zones = self.ec2_client.describe_availability_zones()
         return list(map(lambda x: x['ZoneName'], zones['AvailabilityZones']))
+    
+@dataclass
+class SecurityGroup(Bootstrappable):
+    # Inputs
+    name: str
+    description: str
+    vpc_id: str
+
+    # Outputs
+    group_id: str = field(init=False)
+    arn: str = field(init=False)
+
+    def __post_init__(self):
+        self.name = resources.random_suffix_name("sg-", 24)
+        self.description = resources.random_suffix_name("description-", 34)
+    
+    @property
+    def ec2_client(self):
+        return boto3.client("ec2", region_name=self.region)
+
+    @property
+    def ec2_resource(self):
+        return boto3.resource("ec2", region_name=self.region)
+
+    def bootstrap(self):
+        """Creates security group with an auto-generated name and description.
+        """
+        vpc = self.ec2_resource.VPC(self.vpc_id)
+        group = vpc.create_security_group(
+            Description=self.description,
+            GroupName=self.name,
+        )
```

Comment:
I think we need to set the vpc ID here, other wise it's gonna be created against the default VPC. https://aws-controllers-k8s.github.io/community/reference/ec2/v1alpha1/securitygroup/#spec

Reply:
vpc ID is not required if i'm creating it using this: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/vpc/create_security_group.html

Reply:
ohhh you're right!